apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -151,6 +152,7 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
                                                 final TypeDescription typeDescription,
                                                 final ClassLoader classLoader,
                                                 final JavaModule module) {
+            JVMUtil.getCurrentClassloaderList().add(classLoader);
```

Comment:
1. `JVMUtil` should be renamed to `LoadedLibraryCollector`
2. `getCurrentClassloaderList().add` should be changed as `registerClassLoader`

Reply:
getCurrentClassloaderList + add would end in OOM. Keep using `ArrayList#add`? Would not this cause OOM?

Reply:
I will optimize these places.